inst/tests/tests.Rraw

```diff
@@ -11888,8 +11888,8 @@ test(1774.17, as.data.table(x, na.rm='a'), error="'na.rm' must be scalar")
 
 # verify print.keys works
 DT1 <- data.table(a = 1:3, key = "a")
-test(1775.1, capture.output(print(DT1, print.keys = TRUE)),
-     c("Key: <a>", "   a", "1: 1", "2: 2", "3: 3"))
+test(1775.1, DT1, options = c(datatable.print.keys = TRUE),
```

Comment:
I guess this changes the test slightly, I wonder if it's worth doing something like "check `substitute(x) %iscall% "print"` when `output=` is present, and if so, don't run a simple `print()`" to allow cases where we want to check `print()`'s own behavior directly.

Reply:
It only broke 5 other tests, some of which I already had to touch in #6808, so why not.